source/Engine.cpp

```diff
@@ -837,8 +837,18 @@ void Engine::Step(bool isActive)
 		targetSwizzle = target->GetSwizzle();
 		info.SetString("mission target", target->GetPersonality().IsTarget() ? "(mission target)" : "");
 
+		// Only update the "active" state shown for the target if it is
+		// in the current system and targetable, or owned by the player.
 		int targetType = RadarType(*target, step);
-		info.SetOutlineColor(GetTargetOutlineColor(targetType));
+		if(targetType == Radar::BLINK)
+			info.SetOutlineColor(GetTargetOutlineColor(targetType));
+		else if((target->GetSystem() == player.GetSystem() && target->IsTargetable()) || target->IsYours())
```

Comment:
```suggestion
		else if(target->GetSystem() == player.GetSystem() && target->IsTargetable() || target->IsYours())
```

Reply:
I know the compiler doesn't see a difference here, but I think people will be greatly helped by these brackets.